source/SH3/system/msg/msg.cpp

```diff
@@ -0,0 +1,40 @@
+/** @file
+ *  Implementation of msg.hpp
+ *
+ *
+ *  @copyright 2016-2017  Palm Studios
+ *
+ *  @date 16-1-2017
+ *
+ *  @author Jesse Buhagiar
+ */
+#include "SH3/SH3/system/msg/msg.hpp"
+
+using namespace sh3_system;
+
+void sh3_msg::Load(sh3_arc_vfile& file)
+{
+
+    std::uint8_t c;
+    std::uint8_t c2;
+    sh3_arc_vfile::read_error e;
+
```

Comment:
Sure thing, we'll have to revisit this one, hopefully when we can actually get something on the screen. 

I'm not entirely certain how the original game actually tells what message to load, as all the messages are stashed in the file... I (and Mike) think that one of the unknown file types contains some kind of interaction data, which tells which ID to render.